tutorials/webpack/alias.md

```diff
@@ -0,0 +1,63 @@
+Aliasing is webpack's handy way to shave time and keystrokes
+off importing frequently used modules. You will need the
+`path` npm module as it is how you will tell webpack where
+to look for those specific files. Using the resolve object
+as the parent object and the alias property as the child object,
+you define the grandchildren property names to which you will use
+for your frequently imported modules. Here is an example below:
+
+```javascript
+const path = require('path');
+
+module.exports = {
+    resolve: {
+        alias: {
+            Library: path.resolve(__dirname, 'root/library/'),
+            Single: path.resolve(__dirname, 'root/test.js')
+        }
+    }
+}
+```
+
+So now when you want to import a file from the library module, you use
+`import {file} from 'Library/fileLocation` or if you include the file
+with the alias, `import {test} from 'Single'`.
+
+# Using alias as a boolean
+
+if for whatever reason you needed webpack to ignore something,
```

Comment:
Caps. Also, there's a big reason to use a boolean alias: if you're already loading the library via CDN. Similar to the use case for [Webpack externals](https://masteringjs.io/tutorials/webpack/externals). I don't know if there's a reason to prefer externals over aliases, can you take a look?

Reply:
Took a look, can't find any discussion about the topic so I guess it comes down to preference.